feature/calendar/src/main/kotlin/team/noweekend/feature/calendar/model/DegreeUIModel.kt

```diff
@@ -0,0 +1,51 @@
+package team.noweekend.feature.calendar.model
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Immutable
+import androidx.compose.ui.graphics.Color
+import team.noweekend.core.design.system.foundation.theme.NWKTheme
+import team.noweekend.core.resource.NWKDrawableResource.Degree100
+import team.noweekend.core.resource.NWKDrawableResource.Degree25
+import team.noweekend.core.resource.NWKDrawableResource.Degree50
+import team.noweekend.core.resource.NWKDrawableResource.DegreeAnnualLeave
+import team.noweekend.core.resource.NWKStringResource.DegreeCardAnnualLeaveTitle
+import team.noweekend.core.resource.NWKStringResource.DegreeCardTitle
+
+@Immutable
+data class DegreeUIModel(
+    val degree: Int,
+    val isAnnualLeave: Boolean,
+) {
+    companion object {
+        @DrawableRes
+        fun DegreeUIModel.getResourceImage(): Int {
```

Comment:
@014967 그러면 컴패니언 안쓰고 그냥 멤버 함수로 놔도 괜찮을거 같은뎅?

```suggestion
@Immutable
data class DegreeUIModel(
    val degree: Int,
    val isAnnualLeave: Boolean,
) {
        @DrawableRes
        fun DegreeUIModel.getResourceImage(): Int {
        ...
        }
}
```

Reply:
@SEO-J17 
[3c2f8c8](https://github.com/mash-up-kr/NoWeekend-Android/pull/66/commits/3c2f8c83a539d7b9c82a0114031e9b2629fb317f)

지금 상황에서는 comp object의 이점은 없는거 같아